docs/modding/keyvalues.md

```diff
@@ -0,0 +1,12 @@
+# Changing keyvalues (weapons)
+
+Do keep in note that this is usually changed server-side.
+
+To actually change the keyvalues of weapons you'd have to name them appropriately inside a mod folder\
+For example: `R2Northstar\mods\Northstar.Custom\keyvalues\scripts\weapons`\
+You can find all the weapon keyvalues [here](https://github.com/BigSpice/TitanFall_2_Weapon_Skin_Modding/tree/main/Weapon_Scripts)
+
+- Mod Examples:
+  - [Northstar.Custom](https://github.com/R2Northstar/NorthstarMods/tree/main/Northstar.Custom)
+  - [Unholy Trinity](https://github.com/xamionex/xamionex.UnholyTrinity)
+  - [Kraber9k](https://github.com/Steveplays28/kraber9k/tree/main/Steveplays.Kraber9K)
```

Comment:
Seems to be missing in `SUMMARY.md`.

Reply:
I didn't mean to add keyvalues but I had spare time, but i felt as though it should be expanded before I add it into SUMMARY